docs/_interface/consensus-clients.md

```diff
@@ -0,0 +1,143 @@
+be coupled to another piece of software called a ["consensus client"][con-client-link]. For Geth users that 
+intend to continue to run full nodes after The Merge, it is sensible to start running a consensus client now, 
+so that The Merge can happen smoothly. There are five consensus clients available, all of which connect to Geth in the same way. 
+
+This page will outline how Geth can be set up with a consensus client in advance of The Merge (or to interact with an alread-merged testnet).
+
+
+## Configuring Geth
+
+Geth can be downloaded and installed according to the instructions on the 
+[Installing Geth](/docs/install-and-build/installing-geth) page. In order to connect to a consensus client,
+Geth must expose a port for the inter-client RPC connection. 
+
+The RPC connection must be authenticated using a `jwtsecret` file. This is created and saved 
+to `<datadir>/geth/jwtsecret` by default but can also be created and saved to a custom location or it can be
+self-generated and provided to Geth by passing the file path to `--authrpc.jwtsecret`. The `jwtsecret` file 
+is required by both Geth and the consensus client.
+
+The authorization must then be applied to a specific address/port. This is achievd by passing an address to
+`--authrpc.addr` and a port number to `--authrpc.port`. It is also safe to provide either `localhost` or a wildcard
+`*` to `--authrpc.vhosts` so that incoming requests from virtual hosts are accepted by Geth because it only 
+applies to the port authenticated using `jwtsecret`. 
+
+The Merge itself will be triggered using a terminal total difficulty (TTD). The specific value for the TTD has not yet
+been decided. When it is decided, Geth needs to know what it is in order to merge successfully. This will most likely be 
+included in a new release, so Geth will have to be stopped, updated and restarted in advance of The Merge.
+
+A complete command to start Geth so that it can connect to a consensus client looks as follows:
+
+```shell
+geth --authrpc.addr localhost --authrpc.port 8551 --authrpc.vhosts localhost --authrpc.jwtsecret /tmp/jwtsecret
+```
+
+
+## Consensus clients
+
+There are currently five consensus clients that can be run alongside Geth. These are:
+ 
+[Lighthouse](https://lighthouse-book.sigmaprime.io/): written in Rust
+ 
+[Nimbus](https://nimbus.team/): written in Nim
+ 
+[Prysm](https://docs.prylabs.network/docs/getting-started/): written in Go
+ 
+[Teku](https://pegasys.tech/teku): written in Java
+ 
+There is currently a [client diversity][client-div-link] issue where a large dominance of Prysm clients poses a risk 
+to the health of the network. In response to the initial drive to even out the client diversity many Prysm nodes switched 
+to Lighthouse to the extent that it now also has a problematic market share. It is therefore recommended to use a minority 
+client such as Teku or Nimbus. 
```

Comment:
I think we should avoid information which is likely to to stale in months/years into the docs. Let's keep it more technical. I'd skip this section

Reply:
This content will need to be updated within months anyway because it is written in anticipation of the merge. 
I will remove any information you don't like - of course - but if the diversity situation does indeed change in the next months it will be because this type of content exists in resources like this - imho that would be worth updating a page in the docs for.

How about if it is less specific about the recommended clients (e.g. "It is therefore recommended to use a minority client such as Teku or Nimbus" ---> "Therefore, consider running a minority consensus client"?

Reply:
removed almost all of this, just kept a link to clientdiversity.org in single sentence.